src/main/java/com/neuvector/Scanner.java

```diff
@@ -329,6 +341,63 @@ private static String maskCredential(String message, String credential){
         return message.replace(credential, "******");
     }
 
+    static String[] getDockerGroupCmdArgs(String scanReportPath) {
+        String[] cmdGroupArgs = {"", ""};
+        if (!scanResultsFileExist(scanReportPath) || !isRootFile(scanReportPath)) {
+            cmdGroupArgs[0] = "-u";
+            cmdGroupArgs[1] = executeCommand("id -u");
+            return cmdGroupArgs;
```

Comment:
This line is not needed.

Reply:
59ca9401697b41827c7e1b3454172bf158c472c8